Makefile

```diff
@@ -21,3 +21,10 @@ include release-tools/build.make
 sanity-test: build
 	./test/sanity/run-test.sh
 
+.PHONY: deploy-kind
+deploy-kind:
+	test/utils/deploy-kind.sh
+
```

Comment:
How about using a Github Action for KinD to save on the boilerplate? - https://github.com/marketplace/actions/kind-kubernetes-in-docker-action

Here's an example - https://github.com/linkerd/linkerd2-conformance/blob/main/.github/workflows/kind.yaml